extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayDataSourceRuntimeConfig.java

```diff
@@ -187,4 +187,11 @@ public static final FlywayDataSourceRuntimeConfig defaultConfig() {
      */
     @ConfigItem
     public boolean validateMigrationNaming;
+
+    /**
+     * Ignore migrations during validate and repair according to a given list of patterns.
+     * Patterns are comma separated.
```

Comment:
I think it is important to mention that when this configuration is set, the `ignoreFutureMigrations` and `ignoreMissingMigrations` settings are ignored